airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -44,17 +44,24 @@ class BigQueryHook(GoogleCloudBaseHook, DbApiHook, LoggingMixin):
     """
     conn_name_attr = 'bigquery_conn_id'
 
-    def __init__(self, bigquery_conn_id='bigquery_default', delegate_to=None):
+    def __init__(self,
+                 bigquery_conn_id='bigquery_default',
+                 use_legacy_sql=True,
+                 delegate_to=None):
```

Comment:
Could you move the `use_legacy_sql` to the end of the parameter list? This will have a smaller impact on the API.

Reply:
@Fokko thanks - I've updated with that change